include/bsp/iotc_bsp_tls.h

```diff
@@ -19,25 +19,24 @@
 
 /**
  * @file iotc_bsp_tls.h
- * @brief Implement Transport Layer Security (TLS).
+ * @brief Implements Transport Layer Security (TLS).
  *
- * Reference implementations are in the <a href="~src/bsp/tls"><code></code></a>
- * directory. All TLS functions are non-blocking operations.
+ * @details Reference implementations are in the
+ * <a href="../../../src/bsp/tls"><code></code></a> directory. All TLS functions are
```

Comment:
typo: put the word "directory into the <code> block.

Reply:
Done